scripts/openstack-update.sh

```diff
@@ -91,6 +52,8 @@ if [ "${FAKE_UPDATE}" != "false" ]; then
     OPENSTACK_DEPLOYED_VERSION=$(oc get -n $NAMESPACE $OPENSTACK_VERSION_CR --template={{.spec.targetVersion}})
     sed -i $OUTFILE -e "s|value: $OPENSTACK_DEPLOYED_VERSION|value: $OPENSTACK_VERSION|"
 
+    ${UPDATE_ARTIFACT_DIR}/update_event.sh Applying Fake Update CR
```

Comment:
What is creating update_event.sh file? Is this file created by ci-framework?
I ran this make` make openstack_update_run` and got error: 
`update_event.sh': No such file or directory` and file is not existing on my env.